config/routes.rb

```diff
@@ -64,6 +64,7 @@
         patch :update_response
         patch :take_down
       end
+      resource :petition_details, :only => [:show, :update], as: :petition_details, controller: :petition_details
```

Comment:
I think you could drop these: `, as: :petition_details, controller: :petition_details` from the resource definition